src/main/scala/com/avast/scala/hashes/circe/package.scala

```diff
@@ -5,11 +5,11 @@ import io.circe._
 
 package object circe {
   implicit val Sha256Decoder: Decoder[Sha256] = prepareDecoder(Sha256(_))
-  implicit val Sha256Encoder: Encoder[Sha256] = Encoder.encodeString.contramap((s: Sha256) => s.toString)
+  implicit val Sha256Encoder: Encoder[Sha256] = Encoder.encodeString.contramap((s: Sha256) => s.toBase64String)
```

Comment:
Maybe we could also provide some sub-object with the original hex-encoded behavior to provide some level of compatibility.